mbus-api/src/v0.rs

```diff
@@ -322,8 +334,8 @@ macro_rules! bus_impl_string_id {
     };
 }
 
-bus_impl_string_id!(NodeId, "UUID of a mayastor node");
-bus_impl_string_id!(PoolId, "UUID of a mayastor pool");
+bus_impl_string_id!(NodeId, "ID of a mayastor node");
+bus_impl_string_id!(PoolId, "ID of a mayastor pool");
```

Comment:
Is this intended? Is it appropriate that the other lines are `UUID` still?

Reply:
Yes that was an oversight from a previous change, the Pool and Node use ID's and the Replica and Nexus use UUID's.

Reply:
Okay!